test/e2e/extensions/high_availability_test.go

```diff
@@ -41,16 +41,22 @@ func TestGameServerHealthyAfterDeletingPodWhileOneExtensionsDown(t *testing.T) {
 
 	list, err := getAgoneseExtensionsPods(ctx)
 	logger.Infof("Length of pod list is %v", len(list.Items))
+	for i := range list.Items {
+		logger.Infof("Name of extensions pod %v: %v", i, list.Items[i].ObjectMeta.Name)
+		logger.Infof("Host IP %v", list.Items[i].Status.HostIP)
+		logger.Infof("Pod IPs %v", list.Items[i].Status.PodIPs)
+	}
 	assert.NoError(t, err, "Could not get list of Extension pods")
 	assert.Greater(t, len(list.Items), 1, "Cluster has no Extensions pod or has only 1 extensions pod")
 
-	logger.Info("Removing one of the Extensions Pods")
+	logger.Infof("Removing one of the Extensions Pods: %v", list.Items[1].ObjectMeta.Name)
 	deleteAgonesExtensionsPods(ctx, t)
+	time.Sleep(5 * time.Second)
```

Comment:
At this point, what's the state we are waiting for? 

If we are waiting for a state, we can use a `require.Eventually(t, ...)` to poll until a state is met.

Are we waiting for the pod to be refreshed, or are we actively attempting to see if we can break HA reliability?

Reply:
We are waiting for the pod to be delete and have the number of extensions pod back to 2.